appsignals-tests/contract-tests/src/test/java/software/amazon/opentelemetry/appsignals/test/awssdk/base/AwsSdkBaseTest.java

```diff
@@ -35,16 +38,26 @@
 import software.amazon.opentelemetry.appsignals.test.utils.SemanticConventionsConstants;
 
 public abstract class AwsSdkBaseTest extends ContractTestBase {
+  protected static final Logger LOGGER = Logger.getLogger(AwsSdkBaseTest.class.getName());
+
+  static {
+    ConsoleHandler handler = new ConsoleHandler();
+    handler.setLevel(Level.INFO);
+    LOGGER.addHandler(handler);
+    LOGGER.setLevel(Level.INFO);
+    LOGGER.setUseParentHandlers(false);
+  }
```

Comment:
Why is this needed

Reply:
This was used for debugging purpose. Removed it.